Bottom: dfec70d648229122a223c74fd6c162bb7057df14
Top:    31af2c38b30cc7004b87c3d8c3adf4afc84f8df4
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-07-11 12:48:08 -0700

Refresh of incl-com

---

diff --git a/Makefile.am b/Makefile.am
index b271b45..4a8b9bc 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -124,7 +124,7 @@ man_MANS = \
 	man/rsocket.7
 
 EXTRA_DIST = src/cma.h src/indexer.h src/librdmacm.map \
-	librdmacm.spec.in $(man_MANS)
+	examples/common.h librdmacm.spec.in $(man_MANS)
 
 dist-hook: librdmacm.spec
 	cp librdmacm.spec $(distdir)
